pkg/storaged/client.js

```diff
@@ -505,7 +505,15 @@ function init_model(callback) {
     Promise.allSettled([client.manager.wait(),
         client.mdraids.wait(), client.vgroups.wait(), client.drives.wait(),
         client.blocks.wait(), client.blocks_ptable.wait(), client.blocks_lvm2.wait(), client.blocks_fsys.wait()
-    ]).then(() => {
+    ]).then(results => {
+        // we at least need the manager object; if it doesn't exist, wait for the next proxy onchanged
+        if (results[0].status !== 'fulfilled') {
+            console.warn("init_model(): udisks manager proxy failed:", JSON.stringify(results[0].reason));
+            client.features = false;
```

Comment:
This could just check `client.manager.valid`, which actually happens in enable_udisks_features.

Reply:
Right, it could. Not sure if it's more robust to rely on that or on the .wait() promise. I'm happy to change this.